.travis.yml

```diff
@@ -1,15 +1,11 @@
-#
-# This file has been generated by the Ibuildings QA-Tools
-#
-# Any custom Travis-ci configuration be added here
-# This file should be added to your project repository
-#
-
 language: php
 
-php:
-  - 5.6
-  - 7.0
+matrix:
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION="3.4.*"
+    - php: 7.2
+      env: SYMFONY_VERSION="4.3.*"
```

Comment:
Should we use 4.4 as that will become the next LTS?

Reply:
I've considered this. But 4.4 is not yet released, and I think we can wait until the next update oppurtunity.

Reply:
> I find it very tricky that integration tests are missing to test the used dependencies.

Please vent this on the [Pivotal ticket](https://www.pivotaltracker.com/story/show/169838423) I opened so we can address this when we are starting work on that

> I saw some dependencies which couldn't be resolved in:
> `\Surfnet\StepupBundle\Controller\ExceptionController`.
> `use Symfony\Component\Security\Core\Exception\AccessDeniedException;`
> `use Symfony\Component\Security\Core\Exception\AuthenticationException;`
> 
> See for yourself if you want to fix them as you''ll have the best feeling about this as this was encountered earlier on with the `stepup-saml-bundle`

I don't think we should fix this in the bundle. The fix would be to require symfony/symfony as a Composer requirement. But that implicitly already is the case. You would not be able to use this bundle on an application that is not of the Symfony ecosystem.